arcgis-ios-sdk-samples/AppDelegate.swift

```diff
@@ -177,4 +177,6 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UISplitViewControllerDele
 extension UIColor {
     // Also used as global tint/accent color.
     class var accentColor: UIColor { return UIColor(named: "AccentColor")! }
+    // The translucent background color for status labels.
+    class var statusLabelBackgroundColor: UIColor { return UIColor(named: "statusLabelBackgroundColor")! }
```

Comment:
It looks like this property isn't being used. Am I missing something?

Reply:
I will need this color to replace (unify) many status label's background color in each category. I add it now so that we can branch from `v.next` to update Dark Mode for multiple categories in parallel, after this PR is merged.

If I add this color in a category's PR where it is first needed, it means we have to wait until that category's PR is merged before we can use this color in other categories.

What do you think?

Reply:
I'm a big fan of not introducing code that isn't actually used yet. Plans change all the time and if the code ends up not getting used, it becomes a liability rather than an asset. However, I understand why you added this property now and I think that makes sense.

Reply:
Thanks!